docs/src/pages/theming/color-schemes.mdx

```diff
@@ -95,7 +95,7 @@ import { useMantineColorScheme } from '@mantine/core';
 
 function Demo() {
   const { colorScheme, setColorScheme } = useMantineColorScheme({
-    keepTransition: true,
+    keepTransitions: true,
```

Comment:
Must be updated on line 90 as well

Reply:
https://github.com/mantinedev/mantine/pull/5450/commits/2a2a97d8e2cf5b3f7bda8076bba7b920fb392fda